rqt_top/src/rqt_top/node_info.py

```diff
@@ -32,7 +32,7 @@
     from socket import error
 except ImportError:
     from xmlrpclib import ServerProxy
-    from xmlrpclib.socket import error
+    import xmlrpclib
```

Comment:
This would certainly not work since the type `error` is not known in the code below: https://github.com/ros-visualization/rqt_common_plugins/pull/446/files#diff-a0ffa76f7d85c48cf9f70fec2f2d6cd5R56